docs/asciidoc/advanced-contextPropagation.adoc

```diff
@@ -0,0 +1,38 @@
+[[context.propagation]]
+= Context-Propagation Support
+
+Since 3.5.0, Reactor-Core embeds support for the `io.micrometer:context-propagation` SPI.
+This library is intended as a means to easily adapt between various implementations of the concept of a Context, of which
+`ContextView`/`Context` is an example, and between `ThreadLocal` variables as well.
+
+`ReactorContextAccessor` allows the Context-Propagation library to understand Reactor `Context` and `Contextview`.
+It implements the SPI and is loaded via `java.util.ServiceLoader`.
+No user action is required, other than having a dependency on both reactor-core and `io.micrometer:context-propagation`. The `ReactorContextAccessor` class is public but shouldn't generally be accessed by user code.
+
+On top of that, Reactor-Core 3.5.0 also introduces operators that transparently deal with `ContextSnapshot`s if the library is available at runtime, for users' convenience.
+
+== `contextCapture` Operator
+
+This operator is a convenient alternative to `contextWrite` to perform a `ContextSnapshot#capture` and then to use that snapshot to populate the Reactor `Context`.
```

Comment:
I'd suggest first explaining the intent and then additional context. It's easier to understand first what reactor does without having to dive into `context-propagation` API if it is not necessary (and I understand this API is serving such a purpose). So I'd start with saying that if there are thread locals at subscription time that you need in upstream operators in reactor context, use this operator, but first register in `ThreadLocalAccessor`. I think it's easier to digest starting with the intent and then elaborating on the reason / implementation motivations.

Reply:
missed that one. will reword that part a bit in the next commit.